relengapi/blueprints/slaveloan/static/slaveloan_details.js

```diff
@@ -0,0 +1,16 @@
+function reload_loans() {
```

Comment:
This has the same name as the function in `slaveloan_admin.js` and will overwrite it if both are loaded at once.  It appears to do something different, though.